### DISCUSSION TITLE
Search or filter on labels

### DISCUSSION THREAD

**Discussion by dejantep** [netbox-community/netbox#6120] [Q&A] [answered]
Hello!

There are several places in Netbox where lables are used...inventory items, interfaces, cables
As far as i can see only on cables is possible to filter/search based on label value.
Is there any specific reason why this isnt available across all mentioned models?
Label column is visible by default and it seems counterproductive to not be able to filter based on value

I'm specifically looking to have this on inventory items

**Comment by ryanmerolle** [comment, marked as answer]
https://demo.netbox.dev/dcim/inventory-items/?q=label=Olivia

Basically you can search/filter labels in the model a label is applied, just not globally.
